template-html5-app/src/brightsign/device-info.ts

```diff
@@ -0,0 +1,27 @@
+export interface DeviceInfo {
+    model: string;
+    osVersion: string;
+    bootVersion: string;
+    serialNumber: string;
+    family: string;
```

Comment:
Should this be in a _mock_ folder?

Reply:
I made a __mock__ folder where the raptor.ts file (which defines mock values) is located. I have left device-info.ts and player.ts is still in src. Happy to move them inside the mock folder as well, but those files are not dedicated exclusively to mocks; they define real or mock values.